### ISSUE TITLE
[Grid] Sortable arrow does not have a role attribute and triggers a WCAG violation

### ISSUE THREAD

**Issue by preslavapetrova1** [telerik/kendo-angular#3559]
**Describe the bug**
Currently the aria-label with value "Sortable" gets applied to the span element holding the sort arrow. This element is not focusable and has no role. This issue is flagged as a WCAG violation, since span elements that have an aria-label must also have a role.

![Screenshot 2022-01-26 at 15 15 07](https://user-images.githubusercontent.com/52581229/151169459-94145c5c-1f16-4f18-979a-382bf0286d49.png)

The current scenario can be observed here:
https://stackblitz.com/edit/angular-zxd1yq?file=app/app.component.ts

**Suggestion**
The aria-label can be moved to the operable element in the cell that receives the focus. Additionally, to satisfy the accessibility standards this element should also receive a role="button". 

Tested with Axe tool.
